sequence.puml

```diff
@@ -0,0 +1,24 @@
+@startuml
+participant "Universal Media Server Client"
+participant "API"
+database "API MongoDB"
+
+"Universal Media Server Client" -> API: (1) UMS client requests metadata about a media file
```

Comment:
What are these numbers in brackets for? Can we use autonumber to make that easier to maintain?